app/src/App.vue

```diff
@@ -63,7 +62,8 @@ export default {
       menuItems: [
         { title: "Inventory", path: "/inventory" },
         { title: "Orders", path: "/orders" },
-        { title: "History", path: "/history" }
+        { title: "History", path: "/history" },
```

Comment:
we dont need this, we wont be doing history for the mvp deliverable

Reply:
I didn't add it there, but I'll delete it